int8 pitch
int8 yaw
